File: code/cs/OpcClient.cs
// DOC
using Opc.UaFx.Client;

// DOC
var client = new OpcClient("opc.tcp://localhost:4840/");

// DOC
client.Connect();

// DOC
// Your code to interact with the server.

// DOC
client.Disconnect();

// DOC
using (var client = new OpcClient("opc.tcp://localhost:4840/")) {
    client.Connect();
    // Your code to interact with the server.
}

File: code/vb/OpcClient.vb
'DOC
Imports Opc.UaFx.Client

'DOC
Dim client = New OpcClient("opc.tcp://localhost:4840/")

'DOC
client.Connect()

'DOC
'Your code to interact with the server.

'DOC
client.Disconnect()

'DOC
Using client As New OpcClient("opc.tcp://localhost:4840/")
    client.Connect()
    'Your code to interact with the server.
End Using
